src/services/players/player.hpp

```diff
@@ -67,6 +67,7 @@ namespace big
 
 		rate_limiter m_host_migration_rate_limit{2s, 15};
 		rate_limiter m_play_sound_rate_limit{1s, 10};
+		rate_limiter m_play_sound_rate_limit_tse{1s, 5}; // todo: confirm the correct values
```

Comment:
Now all that remains is to test it properly.

Reply:
@themonsterdev any suggestions on proper rate limiting amounts and time?